test/Microsoft.AspNetCore.Blazor.Build.Test/CodeGenerationTestBase.cs

```diff
@@ -81,6 +81,36 @@ public class MyComponent : BlazorComponent
             CompileToAssembly(generated);
         }
 
```

Comment:
In this file, I notice that in commit d9c69366932a, the line:

    GenerateBaselines = true;

was removed, but then it was put back there in commit f288b05ab6.

Was this deliberate? Shouldn't that flag be `false` normally, so that tests fail if the baselines don't match?

Reply:
yes, and it is false by default. I've debated added a test here that always fails if it's 'true' so that it can't get checked in that way. Maybe I'll go ahead and add it now.

Reply:
What I mean is, it’s explicitly set to true in the checked-in code. Is that deliberate? Sounds like not.

Reply:
Added a test so this will fail if you do it. I also removed the ability to set this PER-test so that our verification will be resiliant. So now it's still easy to iterate on the baselines locally - (ignore the failing tests), but you need to undo it before pushing.